tests/common/test_skearn_model_lists.py

```diff
@@ -69,12 +76,18 @@ def test_get_sklearn_models():
         Ridge,
         TweedieRegressor,
     ]
+
+    expected_neighbors_list = [KNeighborsClassifier]
```

Comment:
excellent! all generic tests work directly for KNN, without changes on the test side? there is nothing really special in the KNN, as compared to the previously existing sklearn models, in terms of how / what we test?

Reply:
lol my bad, it's tests/common/test_skearn_model_lists.py, it's not the generic test file